### ISSUE TITLE
Please report hyperparameters for different data sets, thank you

### ISSUE THREAD

**Issue by skeletonNN** [declare-lab/BBFN#2]

**Comment by Clement25** [comment]
Sorry we didn't have a full record of all hyper-parameters, but you can fix the learning rate for MOSI at 1e-4 and MOSEI at 1e-5 then tune the model. The result should be closed to what we reported.

**Comment by skeletonNN** [comment]
Unfortunately, I adjusted the parameters and the paper should have referenced mult-related code, but I tried many ways and the best F1 value I could get was 79. At the same time, I restored all the original parameters, and only adjusted the learning rate, which was terrible, the effect was very bad.

MAE:  1.1489726
Correlation Coefficient:  0.49840130158778106
mult_acc_7:  0.2594752186588921
mult_acc_5:  0.2725947521865889
F1 score:  0.6971813646707895
Accuracy:  0.698170731707317

**Comment by Clement25** [comment]
> 

Hi, you may forget to add the option 'use_bert' to enable the BERT encoder.
